src/messages/manifestjson.js

```diff
@@ -94,6 +102,14 @@ export const MANIFEST_OPTIONAL_PERMISSIONS = {
   file: MANIFEST_JSON,
 };
 
+export const MANIFEST_HOST_PERMISSIONS = {
+  code: 'MANIFEST_HOST_PERMISSIONS',
+  message: i18n._('Invalid host permission.'),
+  // TODO(#3893): link host_permissions MDN doc page here only we have created it.
```

Comment:
I like to use actual links because I can click them in my IDE (vim). Not a big deal, though.